mondrian/pom.xml

```diff
@@ -164,6 +164,12 @@
       <version>${mysql-connector-java.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Since we don't include all the other possible jdbc drivers, I think we should leave this out.  Readme suggests defining a profile in settings.xml.  I'd stick with that

Reply:
fixed - removed changes to pom.xml.